_posts/2019-10-30-cockpit-206.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Cockpit 206 and Podman 10
+author: Gundersanne
+date: 2019-10-30
+tags: cockpit linux
+slug: cockpit-206
+category: release
+summary: Cockpit with machines, firewall, and login page improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.
+
+### Machines: Network interface deletion
+
+Network interfaces can now be removed from virtual machines.
+
+![Machines NIC deletion](/images/machines-nic-deletion.png)
+
+### login: Enable administration mode by default
+
+For operations that change system state and for troubleshouting Cockpit needs
+administrative rights, so make reusing the password for privileged tasks the
+default.
+
+![login reuse password checkbox is checked](/images/login-reuse-password-checked.png)
+
+### Firewall: Prevent accidental deletion
+
+Deleting zones and services now require 2 clicks minimum.
+
+In addition to making the interface less volatile, Cockpit will warn you when
+you're removing a service or a zone which could impact your current connection.
+
+![Firewall removing a zone warns you](/images/firewall-removing-zone-warning.png)
+
+Furthermore the cockpit service is now added automatically when you add a zone
+via cockpit. That way it's impossible to block yourself out when adding a zone.
+
+### Podman: User containers
+
+The Cockpit Podman page now has support for user containers. These do not
+require administrative privileges to run.
+
+![Cockpit podman user containers](/images/podman-user-containers.png){:.screenshot}
+
+### Try it out
+
+Cockpit 206 is available now:
+
+ * [For your Linux system](https://cockpit-project.org/running.html)
+ * [Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/206)
+ * [Fedora 31](https://bodhi.fedoraproject.org/updates/FEDORA-2019-3f48ea1e16)
+ * [Cockpit Podman](https://bodhi.fedoraproject.org/updates/FEDORA-2019-f72585793e)
```

Comment:
Likewise,  split this for podman  and  cockpit, and we release both of them to F30 and F31, so four bodhi links in total.